src/actions/estimateFee.ts

```diff
@@ -0,0 +1,34 @@
+import { Keypair, PublicKey } from '@solana/web3.js';
+import { Transaction } from '@metaplex-foundation/mpl-core';
+import { Connection } from '../Connection';
+
+interface Params {
+  connection: Connection;
+  feePayer: PublicKey;
+  txs: Transaction[];
+  signers?: Keypair[];
+}
+
+type EstimateFee = (params: Params) => Promise<number>;
```

Comment:
I'd use `BN` for the returned transaction fee
And please leave a comment that you return fee in lamports

Reply:
What do you think about it?
https://github.com/metaplex-foundation/js/pull/122/files#diff-77f355a8bdb1ffc2f598fe6eb8153c7fca90aa361b947abdae2941335abb312dR13